TODO.md

```diff
@@ -7,6 +7,18 @@ Scratchpad for random ideas of things that need done.
 * Verify RDP enable
 * Linux repos
 * Whole buncha stuff for profile::cfcc::camper
+Charlie's $0.02
+
+* jdk8 > jdk10
+* remove eclipse
```

Comment:
I'd be curious to get @piticent123 thoughts about use/nonuse of Eclipse. I've found it on a few camp machines.

Reply:
IDEA  Community is free, open source, and better. I thought for a second maybe eclipse might make sense for android, but that ship sailed years ago and its all IDEA based now

Reply:
It would also be more consistent across python, web, php, or whatever other platform we chose, except maybe ios, although I think they have an IDE for that too.

Reply:
I agree with charlie. the eclipse on camp machines was probably still from when we used google code/subversion 😛 i always have my kids install idea

Reply:
👍 Be gone with it!